.github/workflows/merlin.yml

```diff
@@ -258,47 +260,46 @@ jobs:
           path: merlin.${{ needs.create-version.outputs.version }}.tar
           retention-days: ${{ env.ARTIFACT_RETENTION_DAYS }}
 
-  build-batch-predictor:
+  build-push-batch-predictor-base:
```

Comment:
What's the motivation for combining the build and publish steps? With this, we will be publishing the base images for every single commit to the PR / main, unlike previously, when the images are only pushed if the e2e succeeds. Shall we keep them separate as before?

Reply:
-> What's the motivation for combining the build and publish steps?
No strong opinion here. It was to fasten my development iteration. I'll separate it again.